spec/spec_helper.rb

```diff
@@ -16,11 +17,17 @@
 require 'rspec/rails'
 require 'rspec/autorun'
 require 'capybara/rspec'
+require 'vcr'
 
 # Requires supporting ruby files with custom matchers and macros, etc,
 # in spec/support/ and its subdirectories.
 Dir[Rails.root.join('spec/support/**/*.rb')].each { |f| require f }
 
+VCR.config do |c|
+  c.cassette_library_dir     = 'spec/cassettes'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.